fedora/cinc/Dockerfile

```diff
@@ -18,7 +18,7 @@
 #      $ docker run -d --privileged openshift/dind
 #
 
-FROM fedora:30
+FROM fedora:31
```

Comment:
@numansiddique : Please add to line 72 the following package

dnf install python3-pip

Reply:
Done